artman/tasks/protoc_tasks.py

```diff
@@ -316,55 +59,57 @@ def validate(self):
 
 
 class ProtocCodeGenTaskBase(task_base.TaskBase):
-    default_provides = 'grpc_code_dir'
+    default_provides = "grpc_code_dir"
 
     def _execute_proto_codegen(
-            self, language, src_proto_path, import_proto_path,
-            output_dir, api_name, api_version, organization_name,
+            self, output_dir, language, src_proto_path, import_proto_path,
```

Comment:
Nit: why change the parameter ordering? Also, this should be renamed (probably to `pkg_dir`), because it is not called with the value of `output_dir` that is provided to the task.